pages/schedule/index.tsx

```diff
@@ -154,44 +154,30 @@ export default function Calendar(props: { scheduleCard: ScheduleEvent[] }) {
     ),
   );
 
-  const changeEventData = (data) => {
-    const startDate = new firebase.firestore.Timestamp(data.startTimestamp._seconds, 0).toDate();
-    const endDate = new firebase.firestore.Timestamp(data.endTimestamp._seconds, 0).toDate();
```

Comment:
Is there a reason we were doing this, or am I okay to just use the included `startDate` and `endDate` fields computed by the API route?

Reply:
This change looks and behaves fine; the `firebase.firestore.Timestamp` stuff was implemented by @nam-t24 in #87 if he has more insight into this